converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Modules/DicomModule.cs

```diff
@@ -41,6 +41,10 @@ public void Load(IServiceCollection services)
         services.AddHttpClient<IDicomWebClient, DicomWebClient>(sp =>
             {
                 sp.BaseAddress = dicomWebConfiguration.Endpoint;
```

Comment:
nit: we should probably use the `IServiceProvider` to fetch the options instead of using the value directly from the config. That way users can modify the options before they're finally resolved:
```csharp
IOptions<DicomWebConfiguration> config = sp.GetRequiredService<IOptions<DicomWebConfiguration>>()
```

Reply:
lol why not do this?

Reply:
updated